### ISSUE TITLE
Have a reporter that combines the benefits of FindingsReporter and LiteFindintsReporter

### ISSUE THREAD

**Issue by gfreivasc** [detekt/detekt#4463]
It would be good to have a reporter that combines the easy to navigate layout structure of `FindingsReporter` with the extra details found on `LiteFindingsReporter`.

## Expected Behavior

```kotlin
fun main() {
println("Hello World!")
}
```

It would be nice to have the output being as follows:

```
formatting - 5min debt
        Indentation - [Unexpected indentation (0) (should be 4)] at /Users/gabrielfv/Projects/detekt/detekt-core/src/main/kotlin/io/gitlab/arturbosch/detekt/core/reporting/Reporting.kt:23:1
```

## Current Behavior

For `FindingsReporter`

```
formatting - 5min debt
        Indentation - [] at /Users/gabrielfv/Projects/detekt/detekt-core/src/main/kotlin/io/gitlab/arturbosch/detekt/core/reporting/Reporting.kt:23:1
```

Which omits details about the rule

For `LiteFindingsReporter`

```
/Users/gabrielfv/Projects/detekt/detekt-core/src/main/kotlin/io/gitlab/arturbosch/detekt/core/reporting/Reporting.kt:23:1: Unexpected indentation (0) (should be 4) [Indentation]
```

Which displays the information, but in a rather messy format, especially on large reports.

## Context

Based on #4446.

**Comment by cortinico** [comment]
Fixed by #4464
